### ISSUE TITLE
Add travis_wait-like feature

### ISSUE THREAD

**Issue by ipa-mdl** [ros-industrial/industrial_ci#281]
`travis_wait`  adds some  log message to the output at the end of each job, but might be required for installing a lot of packages or non-verbose build.
Instead we  could include a custom output that will be displayed each 9 minutes, like in the travis script I prepared for [ros-industrial/docker](https://github.com/ros-industrial/docker/blob/master/build.sh#L17).

This would make travis run as long as needed, but there is a little chance that the output (especially the folds) gets mangled.

**Comment by fmessmer** [comment]
I happen to see the timeout message quite a lot recently:
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

Also, for jobs that have been working before.
It seems to happen for both `indigo` and `kinetic` jobs.
Has anything changed in `industrial_ci` or `travis` itself recently that causes the increase in build terminations?

**Comment by ipa-mdl** [comment]
The `industrial_ci ` sporadically show the same issue now, in jobs that should not exceep 3 minutes..

**Comment by ipa-mdl** [comment]
implemented in #324
